packages/eslint-plugin/src/rules/no-deprecated.ts

```diff
@@ -116,17 +116,21 @@ export default createRule({
     function getJsDocDeprecation(
       symbol: ts.Signature | ts.Symbol | undefined,
     ): string | undefined {
-      const tag = symbol
-        ?.getJsDocTags(checker)
-        .find(tag => tag.name === 'deprecated');
+      try {
```

Comment:
Let's try to be as surgical about the try/catch as we can - can we make it so that essentially _only_ the `getJsDocTags()` call is wrapped?